chain/chain/src/test_utils.rs

```diff
@@ -61,7 +61,7 @@ pub struct KeyValueRuntime {
     tries: ShardTries,
     validators: Vec<Vec<ValidatorStake>>,
     validator_groups: u64,
-    num_shards: ShardId,
+    num_shards: u64,
```

Comment:
Please replace back with `ShardId`

Reply:
I believe there is a type called `NumShards`, introduced by @Kouprin

Reply:
@bowenwang1996 I wasn't able to find it and I thought someone deleted it for keeping `ShardId`. CLion restart helped.
++ for `NumShards`.